app/views/performance_reports/show.html.slim

```diff
@@ -37,7 +37,20 @@ table.table
       td
     tr.active
       td= t('.volunteers.active_total')
-      td= @report_content['global']['volunteers']['active']
+      td= @report_content['global']['clients']['active']
+
+    tr
+      td= t('.clients.total')
+      td= @report_content['global']['clients']['total']
+    tr
+      td= t('.clients.new')
+      td= @report_content['global']['clients']['new']
+    tr
+      td= t('.clients.resigned')
+      td
```

Comment:
Do we have info to display here?

Reply:
thats what I was mentioning. Thats the missing number.